src/map/script.cpp

```diff
@@ -13962,7 +14093,8 @@ BUILDIN_FUNC(getinventorylist)
 {
 	TBL_PC *sd;
 	char card_var[NAME_LENGTH], randopt_var[50];
-	int i,j=0,k;
+	int i,j=0,k, maxlen = 256;
+	char* buf = (char*)aMalloc(maxlen * sizeof(char));
```

Comment:
memory leak will be here.

Reply:
no, you did not get me =)
memset will set memory area by nulls, while you need to free memory to the operating system. you need to use aFree(buf) somewhere in your code before exiting the function.